lua/fpp/server/ownability.lua

```diff
@@ -286,13 +286,16 @@ end
 Networking
 ---------------------------------------------------------------------------]]
 util.AddNetworkString("FPP_TouchabilityData")
--- Sends 13 + 8 + 5 + 20 = 46 bits of ownership data per entity
+
+-- Sends MAX_EDICT_BITS(13) + (1, 8) + 5 + 20 = (39, 46) bits of ownership data per entity
+local MAX_PLAYER_BITS = math.ceil(math.log(1 + game.MaxPlayers()) / math.log(2))
```

Comment:
I was struggling quite a bit with the `(1, 8)` notation, and it took me quite a while to find out that it referred to the range of values that `MAX_PLAYER_BITS` can have, depending on `game.MaxPlayers()` returning something between 1 and 128. Here's a suggestion for another comment:


```suggestion
-- MAX_PLAYER_BITS is the minimum amount of bits needed to refer to a player.
-- Given that there's a hard maximum of 128 players on a server. MAX_PLAYER_BITS
-- will hold a value between 1 and 8 inclusive.
local MAX_PLAYER_BITS = math.ceil(math.log(1 + game.MaxPlayers()) / math.log(2))
--
-- Sends MAX_EDICT_BITS(13) + MAX_PLAYER_BITS(between 1 and 8) + 5 + 20 = between 39 and 46 bits of
-- ownership data per entity
```